tasks/destroy.js

```diff
@@ -5,18 +5,22 @@ let spawn = require('shellpromise');
 function task (options) {
 	let app = options.app;
 	let verbose = options.verbose;
-	let promise = Promise.resolve();
-	if (verbose) {
-		promise = promise.then(function () {
 
+	return spawn('heroku info ' + app, {verbose: true }).then(function () {
```

Comment:
Do we still need `verbose` here now, or can we get rid of it?

Reply:
No, we don't! I'll get rid of it.